src/styles/card.scss

```diff
@@ -0,0 +1,156 @@
+@import "variables";
+
+.card {
+  width: 200px;
+  height: 408px;
+  background-color: $background-color;
+  border-radius: 5px;
+  border: solid 1px $border-color;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  transition: all 300ms ease;
```

Comment:
write all of the properties you want to transition, instead of using `transition: all`